django/contrib/admin/static/admin/css/base.css

```diff
@@ -21,10 +21,10 @@ html[data-theme="light"],
     --header-link-color: var(--primary-fg);
 
     --breadcrumbs-fg: #c4dce8;
-    --breadcrumbs-link-fg: var(--body-bg);
+    --breadcrumbs-link-fg: var(--body-loud-color);
```

Comment:
This is better but misses the fact breadcrumb links are still too low-contrast on hover. Perhaps we could proceed with this as-is though.

For hover, my preference would be to simply underline the links so color is no longer an issue.